romancal/source_detection/source_detection_step.py

```diff
@@ -43,7 +43,7 @@ class SourceDetectionStep(RomanStep):
         roundhi = float(default=1.0)  # DAOStarFinder: Upper bound for roundness.
         peakmax = float(default=1000.0)  # Upper limit on brightest pixel in sources.
         max_sources = float(default=None)  # Max number of sources, choosing brightest.
-        scalar_threshold = float(default=None) # Detection threshold to be used 
+        scalar_threshold = float(default=None) # Detection threshold to be used
```

Comment:
Which unit should we use for the threshold? Is it absolute or relative?

Reply:
updated for clarification, its absolute